odml/section.py

```diff
@@ -81,6 +82,8 @@ def __init__(self, name=None, type=None, parent=None,
         self.type = type
         self.parent = parent
 
+        validation.Validation(self)
```

Comment:
Errors in a validation should be printed at this point to inform the user of any problems with this odml object. Otherwise the validation has no effect. Warnings should be ignored for now.